src/app/components/entreprise-form/entreprise-form.component.html

```diff
@@ -0,0 +1,285 @@
+
+          <div class="mb-4">
+            <label for="github" class="block text-sm font-medium text-gray-700">
+              GitHub
+            </label>
+            <input
+              id="github"
+              pInputText
+              formControlName="github"
+              type="url"
+              class="mt-1 block w-full rounded-md border-gray-300 p-2 shadow-md outline-byte-200"
+            />
+          </div>
+
+          <div class="mb-4">
+            <label
+              for="facebook"
+              class="block text-sm font-medium text-gray-700"
+            >
+              Facebook
+            </label>
+            <input
+              id="facebook"
+              pInputText
+              formControlName="facebook"
+              type="url"
+              class="mt-1 block w-full rounded-md border-gray-300 p-2 shadow-md outline-byte-200"
+            />
+          </div>
+        </div>
+
+        <div class="mb-4">
+          <label
+            for="shortDescription"
+            class="block text-sm font-medium text-gray-700"
+          >
+            Description Brève de l'Entreprise
+          </label>
+          <textarea
+            id="shortDescription"
+            pInputTextarea
+            formControlName="shortDescription"
+            class="mt-1 block w-full rounded-md border-gray-300 p-2 shadow-md outline-byte-200"
+          ></textarea>
+        </div>
+
+        <div class="mb-4">
+          <label
+            for="longDescription"
+            class="block text-sm font-medium text-gray-700"
+          >
+            Description Longue de l'Entreprise
+          </label>
+          <textarea
+            id="longDescription"
+            pInputTextarea
+            formControlName="longDescription"
+            class="mt-1 block w-full rounded-md border-gray-300 p-2 shadow-md outline-byte-200"
+          ></textarea>
+        </div>
+
+        <div class="mb-4">
+          <label for="sectors" class="block text-sm font-medium text-gray-700">
+            Secteurs
+          </label>
+          <p-chips
+            id="sectors"
+            formControlName="sectors"
+            separator=","
+            max="10"
+            class="mt-1 block w-full shadow-md outline-byte-200"
+          ></p-chips>
+        </div>
+
+        <div class="mb-4">
+          <label
+            for="technologiesUsed"
+            class="block text-sm font-medium text-gray-700"
+          >
+            Technologies Utilisées
+          </label>
+          <p-chips
+            id="technologiesUsed"
+            formControlName="technologiesUsed"
+            separator=","
+            max="15"
+            class="mt-1 block w-full shadow-md outline-byte-200"
+          ></p-chips>
+        </div>
+
+        <div class="mb-4">
+          <label for="logo" class="block text-sm font-medium text-gray-700">
+            Logo
+          </label>
+          <input
+            id="logo"
+            pInputText
```

Comment:
should be a file jpg/jpeg/png

Reply:
I suggest using URLs instead, as most companies have their logos readily available online. This approach allows contributors to simply copy the image address, streamlining the process without the need to download and upload files. Using URLs also reduces storage requirements and ensures that the most current version of the logo is always displayed, as any updates on the company’s website are automatically reflected.

Reply:
that might be a positive but the potential negatives are worse, we don't want to use a url that might go down or the image might change if the url is not an official one, so i prefer getting the image, it's easier and faster and some people might not even know how to get an image url ... as far as storage, that's not an issue, we'll be using a cdn